rules/sinks/third_parties/sdk/activecampaign/javascript.yaml

```diff
@@ -0,0 +1,13 @@
+
+#  Sink rule for ThirdParty SDK
+#  The id follows a format : "ThirdParties.SDK.<THIRD_PARTY_ORGANISATION>.<SUB_ORGANISATION_IF_APPLICABLE>"
+
+sinks:
+
+  - id: ThirdParties.SDK.ActiveCampaign
+    name: ActiveCampaign
+    domains:
+      - "activecampaign.com"
+    filterProperty: "code"
```

Comment:
Overarching comment - we are using `code` based filtering across all the rules in this file because we are not getting the right CPG nodes ? What is the reason for this ?

Reply:
As these rules are for analytics related sinks and can be called directly using browser `window` object. here is the [example](https://github.com/rudderlabs/rudder-sdk-js/blob/4ad61db9408f37449d62368d192b5d06e36e87ca/packages/analytics-js-integrations/src/integrations/GA/browser.js#L62C9-L62C28), that is difficult to resolve its methodFullName. so I guess we preferred code based filtering. @HiteshMah-Jan correct me if i missed anything.